app/dts/behaviors/soft_off.dtsi

```diff
@@ -6,7 +6,7 @@
 
 / {
     behaviors {
-        /omit-if-no-ref/ soft_off: keymap_soft_off {
+        /omit-if-no-ref/ soft_off: soft_off {
```

Comment:
This had originally been added to reduce the risk of folks adding additional DTS with the same node name accidentally... Perhaps something like `z_so_beh` so it's ever less likely of accidentally getting overriden?

Reply:
Is that really necessary? We don't do that for any other behaviors. Including "behavior" in the node name isn't necessary either, since behaviors have their own lookup that's separate from other node types.

Reply:
Probably overkill, but the only reason this one is a bit different than the others is the fact it is the first behavior that is likely to be used *directly* in board dts or shield overlay files, and if created there, accidentally clash with the "keymap version".

Reply:
It won't hurt to rename it, I guess, I updated the PR